Weights/examples/Weights/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Created by the script cgal_create_cmake_script.
+# This is the CMake script for compiling a CGAL application.
+
+project(Weights_Examples)
+
+cmake_minimum_required(VERSION 3.1...3.15)
+set(CMAKE_CXX_STANDARD 14)
+
+find_package(CGAL QUIET COMPONENTS Core)
```

Comment:
Note that it is now advised to add `required` and remove the `if (CGAL_FOUND)`

Reply:
Thanks. I will fix it.

Reply:
Done.